README.md

```diff
@@ -10,13 +10,13 @@
 
 # The HTML-first framework
 
-Qwik offers the fastest possible page load times - regardless of the complexity of your website. Qwik is so fast because it allows fully interactive sites to load with almost no JavaScript and [pickup from where the server left off](https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/concepts/resumable.mdx).
+Qwik offers the fastest possible page load times - regardless of the complexity of your website. Qwik is so fast because it allows fully interactive sites to load with almost no JavaScript and [pickup from where the server left off](https://github.com/BuilderIO/qwik/blob/main/packages/docs/src/routes/docs/concepts/resumable/index.mdx)
```

Comment:
I think this paragraph was written before we had a docs site. Would you be able to point the url to https://qwik.builder.io/docs/concepts/resumable instead?

Reply:
Both resolved